awx/main/tests/functional/test_credential.py

```diff
@@ -74,12 +74,15 @@
 
 @pytest.mark.django_db
 def test_default_cred_types():
+    import pdb
+
+    pdb.set_trace()
     assert sorted(CredentialType.defaults.keys()) == [
         'aim',
         'aws',
         'azure_kv',
         'azure_rm',
-        'centrify_vault_kv',
+        #        'centrify_vault_kv',
```

Comment:
I had to comment out this line for the test to pass. Any pointers as to why this might be causing the issue ?